source/brailleDisplayDrivers/baum.py

```diff
@@ -162,11 +188,17 @@ def _onReceive(self, data):
 		self._handleResponse(command, arg)
 
 	def _handleResponse(self, command, arg):
+		"""
+		:type command: bytes
+		:type arg: bytes
+		"""
 		if command == BAUM_CELL_COUNT:
+			# Assumption: BAUM_CELL_COUNT command has a single byte unsigned argument.
+			# Value range (0-255)
 			self.numCells = ord(arg)
 		elif command == BAUM_DEVICE_ID:
 			# Short ids can be padded with either nulls or spaces.
-			self._deviceID = arg.rstrip("\0 ")
+			self._deviceID = arg.rstrip(b"\0 ")

```

Comment:
I think we should save this one as a string.

```suggestion
			self._deviceID = arg.rstrip(b"\0 ").decode("latin-1") # Assumption
```